cmake/FindGeode.cmake

```diff
@@ -73,7 +73,7 @@ if(Geode_gfsh_EXECUTABLE)
     # TODO error checking
   else()
     if(var MATCHES "([0-9]+\\.[0-9]+\\.[0-9]+)")
-      set(Geode_VERSION "${CMAKE_MATCH_1}")
+      set(Geode_VERSION ${var})
```

Comment:
What is this all about?  I don't understand why we need a change in the Geode find module...

Reply:
I agree, the group 1 has all the parts we care about in the version.

Reply:
Good catch! No longer need the build number since we're just comparing with first 3 parts.